### ISSUE TITLE
Suggestion: newsletter for feedstock maintainers

### ISSUE THREAD

**Issue by moorepants** [conda-forge/conda-forge.github.io#759]
I maintain a reasonable number of feedstocks but simply don't have time to keep up with the seemingly fast pace changes to the build system. I've been confused about the compiler update PRs, blas update PRs, new CI services, etc. and rarely know how to diagnose things. I would really love to get some kind of newsletter, version change log, etc with some documentation and information showing upcoming or in progress changes so we could have some idea of what we are supposed to do when these PRs occur and especially when they fail to build. Maybe this information is supplied somewhere, if so pointing me to that location or method of getting it would be helpful too. Thanks for all the hard work and I understand I'm asking for someone to do extra work, but maybe a change log for conda forge wouldn't be too much extra.

**Comment by moorepants** [comment]
I do now see that the conda forge docs are very different than the last time I read them. Will review :)

**Comment by croth1** [comment]
I totally agree with you here. The documentation/information strategy of CF is by far not good enough for a project with this number of contributors. Things are moving quickly here, but if you do not track the changes regularly over a longer period it is easy to find oneself completely lost.

I will think a bit more how to tackle this. An active news feed with major changes and where our one can find more information is definitely a very good idea.

**Comment by croth1** [comment]
Has also been suggested by @kkraus14 on gitter: https://gitter.im/conda-forge/conda-forge.github.io?at=5ca23c7425686a7dc3ba8e07.

@conda-forge/core I think we should think about this again.
